validation/input/input_1/input_script_1.py

```diff
@@ -1 +1,50 @@
-print("Hello world!")
+import pytest
+import sys
+import os
+from pathlib import Path
+import time
+
+
+EXPECTED_INPUTS_ENVS = ["INPUT_FOLDER", *(f"INPUT_{i}" for i in range(1, 6))]
```

Comment:
NOTE: when you check this online (i.e. uploading the zip with this file and ther equirements) ... please observe if shows TWICE the logs ... that is my impression

Reply:
yes I saw that too. but I guess this is something to do with osparc. will create an issue there